.vscode/launch.json

```diff
@@ -20,21 +20,11 @@
       "type": "extensionHost",
```

Comment:
I don't know if its worth keeping this second command. As far as I remember this one was created to actually debug it , whereas the last one was only for running it. And now as there is only one single vscode extension, the upper one will do the work of debugging it. Unless there are specific scenarios to run and keep this one.

Reply:
@DanielleCogs Any thoughts on this - was there a reason you wanted to keep this separate data mapper command?

Reply:
I was thinking that this could save us some time by not having to build the other projects at every update! Carlos lmk if that makes sense here